src/system/settings/settings.ts

```diff
@@ -363,7 +363,7 @@ export const Setting: Array<Setting> = [
   },
   {
     key: SettingKeys.Reroll_Target,
-    label: "Reroll Target",
+    label: i18next.t("settings:Reroll Target"),
```

Comment:
Normally we don't use "human writing" for keys but `camelCase`

```suggestion
    label: i18next.t("settings:rerollTarget"),
```